audio_service/lib/audio_service.dart

```diff
@@ -903,19 +927,22 @@ class AudioService {
     _handler.androidPlaybackInfo
         .listen((AndroidPlaybackInfo playbackInfo) async {
       await _platform.setAndroidPlaybackInfo(SetAndroidPlaybackInfoRequest(
-          playbackInfo: playbackInfo._toMessage()));
+        playbackInfo: playbackInfo._toMessage(),
+      ));
     });
     _handler.queue.listen((List<MediaItem>? queue) async {
       if (queue == null) return;
       if (_config.preloadArtwork) {
         _loadAllArtwork(queue);
       }
       await _platform.setQueue(SetQueueRequest(
-          queue: queue.map((item) => item._toMessage()).toList()));
+        queue: queue.map((item) => item._toMessage()).toList(),
```

Comment:
I wouldn't personally put the comma here. Although I don't have a strong preference, I value small diffs more ;-) Unless the change has clear and demonstrable value. This is probably not something I could be convinced there is.

Reply:
this is to be consistent with other uses (see above an below)
> I value small diffs more

i value code readability, that is to me having at least some separation in mess of `)))`, and also having dartfmt wrapping you lines in weird ways like this https://github.com/ryanheise/audio_service/pull/640/files#diff-dea95db8d3d329f3f9c1ad039764dfd5de254ea2a2c600c2d8da3b1a59bc5404L917-L918

yet again, not trying to reconvince you in some of your preferences, explaining why did i make it here

Reply:
Understood, although all code was written in the first place to be readable by its author, and we just want to avoid different people changing formatting back and forth to their own liking. Until we get some lints in place, we can just preserve the existing style as much as possible, imitate it, and spend our efforts elsewhere on solving the problems that app developers are running into.

Regarding the trailing comma:

https://flutter.dev/docs/development/tools/formatting#using-trailing-commas

> The guideline for adding a trailing comma is simple: Always add a trailing comma at the end of a parameter list in functions, methods, and constructors where you care about keeping the formatting you crafted.

Even at that, beauty is in the eye of the beholder on a case by case basis. FYI, I would only generally attempt to format structure like this if there were more than one (named) parameter where those parameters could be aligned at the same indent level, but on a whim I might do it your way, if I like the look of it, on a case by case basis (e.g. to keep the lambda function unbroken on one line).